services/src/test/resources/application.conf

```diff
@@ -2,7 +2,7 @@
       gitlab {
         url = "https://gitlab.com/api/v4/"
         token = "XXXX" //TODO: Change for real GitLab private token
-        defaultFileVersion = "v.0.0.1"
+        defaultFileVersion = "v0.0.1"
         defaultBranch = "master"
       }
   }
```

Comment:
Checker is grumpy about absence of a new line at the end of file

Reply:
Fixed